tcf_website/templates/base/sidebar.html

```diff
@@ -1,35 +1,49 @@
 {% load static %}
 
 <!-- Sidebar -->
-<div class="border-right
-d-flex flex-column justify-content-between" id="sidebar-wrapper">
+<div id="sidebar-wrapper" class="border-right pb-0
+     d-flex flex-column justify-content-between text-center">
   <div class="list-group list-group-flush">
```

Comment:
It would be nice to swap out the `div`s here with `ul` and `li`s since this is a list group, to make it more semantic HTML

Reply:
done